lib/controllers/situations.js

```diff
@@ -44,10 +44,13 @@ exports.create = function(req, res, next) {
     });
 };
 
-exports.simulation = function(req, res, next) {
+exports.simulation = function(req, res) {
     req.situation.simulate(function(err, result) {
-        if (err) return next(err);
-        res.send(result);
+        if (err) {
+            res.status(err.code).send(err);
```

Comment:
Are you 100% sure such an `err` will always have a `code` property? If not, it would be safer to use `err.code || 500` 🙂